src/components/TextInput/index.js

```diff
@@ -228,6 +228,7 @@ class TextInput extends Component {
       onChange: normalizeEventHandler(this._handleChange),
       onFocus: normalizeEventHandler(this._handleFocus),
       onKeyPress: normalizeEventHandler(this._handleKeyPress),
+      onKeyDown: normalizeEventHandler(this._handleKeyDown),
```

Comment:
It looks like we don't need to use `normalizeEventHandler` here or for `onKeyPress`, as we're not using the native event or passing it to any prop handlers within the component.